src/rules/selector-combinator-space/README.md

```diff
@@ -0,0 +1,183 @@
+# selector-combinator-space
+
+Require or disallow a space before and/or after the combinator of selectors.
```

Comment:
A short explanation of what combinators are would help. Also, we should note that this rule does not address the descendent combinator, which is just a space.